cdap-unit-test/src/test/java/io/cdap/cdap/spark/SparkStreamingTestRun.java

```diff
@@ -53,6 +54,7 @@ public class SparkStreamingTestRun extends TestFrameworkTestBase {
   public static final TemporaryFolder TEMP_FOLDER = new TemporaryFolder();
 
   @Test
+  @Ignore
```

Comment:
For tests that we have added an ignore tag, can we go ahead and open a JIRA ticket for them and add as a TODO comment? Preferably logically grouped by efforts it would take to fix. Thanks!

Reply:
Ack.